software/nanoplot/main.nf

```diff
@@ -0,0 +1,46 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process NANOPLOT {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::nanoplot=1.32.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/nanoplot:1.32.1--py_0"
+    } else {
+        container "quay.io/biocontainers/nanoplot:1.32.1--py_0"
+    }
+
+    input:
+    tuple val(meta), path(fastq), path(summary_txt)
+
+    output:
+    tuple val(meta), path("*.html"), emit: html
+    tuple val(meta), path("*.png") , emit: png
+    tuple val(meta), path("*.txt") , emit: txt
+    tuple val(meta), path("*.log") , emit: log
+    path  "*.version.txt"          , emit: version
+
+    script:
+    def software = getSoftwareName(task.process)
+    """
+    NanoPlot \\
+        $options.args \\
+        -t $task.cpus \\
+        --fastq $fastq
+
+    NanoPlot \\
+        $options.args \\
+        -t $task.cpus \\
+        --summary $summary_txt
+
```

Comment:
```suggestion
```

Reply:
I have removed the extra NanoPlot command